Meta/CMake/unicode_data.cmake

```diff
@@ -1,139 +1,124 @@
-set(LOCALE_DATA_CACHE_LOCATION "${CMAKE_BINARY_DIR}/CLDR" CACHE PATH "Download location for CLDR files")
-set(UNICODE_CACHE_LOCATION "${CMAKE_BINARY_DIR}/UCD" CACHE PATH "Download location for UCD files")
+set(UCD_VERSION 14.0.0)
+set(CLDR_VERSION 39.0.0)
 
-set(UNICODE_DATA_URL https://www.unicode.org/Public/13.0.0/ucd/UnicodeData.txt)
-set(UNICODE_DATA_PATH ${UNICODE_CACHE_LOCATION}/UnicodeData.txt)
+set(UCD_PATH "${CMAKE_BINARY_DIR}/UCD" CACHE PATH "Download location for UCD files")
+set(CLDR_PATH "${CMAKE_BINARY_DIR}/CLDR" CACHE PATH "Download location for CLDR files")
```

Comment:
Fwiw the CI scripts and Superbuild.cmake were setting the path for these cached artifacts using the old name (for when you get back to this :) )